src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 57px;
+  z-index: 2;
+`;
+
+export const StyledTopBar = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 56px;
+  background-color: #2a2a2a;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14),
+    0px 1px 2px rgba(0, 0, 0, 0.2);
+  z-index: 3;
+`;
+
+export const StyledHeader = styled.header`
+  position: absolute;
+  width: 176px;
+  height: 24px;
+  left: 69px;
+  top: 16px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 19px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  color: #3c78c8;
+`;
+
+export const StyledControls = styled.div`
+  position: absolute;
+  width: 99px;
+  height: 24px;
+  right: 16px;
+  top: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const StyledIcon = styled.img`
+  position: relative;
+  width: ${props => props.width || '24px'};
+  height: ${props => props.height || '24px'};
+`;
+
+export const StyledGradientLine = styled.div`
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  height: 1px;
+  z-index: 4;
```

Comment:
In reality we don't have 4 layers in Z direction here.
In reality It's all one layer. This line is just a border for the top bar. It's on layer `0`.
Why do we need to specify `z-index` here at all?